tests/foreman/sys/test_katello_certs_check.py

```diff
@@ -145,8 +200,75 @@ def test_positive_validate_katello_certs_check_output(self, cert_setup, cert_dat
             )
         self.validate_output(result, cert_data)
 
+    @pytest.mark.tier1
+    def test_katello_certs_check_output_wildcard_inputs(self, generate_certs, cert_data):
+        """Validate that katello-certs-check generates correct output with wildcard certs.
+
+        :id: 7f9da806-5b23-11eb-b7ea-d46d6dd3b5b2
+
+        :steps:
+
+            1. Get valid wildcard certs from generate_certs
+            2. Run katello-certs-check with the required valid arguments
+               katello-certs-check -c CERT_FILE -k KEY_FILE -r REQ_FILE
+               -b CA_BUNDLE_FILE
+            3. Assert the output has correct commands with options
+
+        :expectedresults: katello-certs-check should generate correct commands
+         with options.
+        """
+        with get_connection() as connection:
+            result = connection.run(
+                'katello-certs-check -c {} -k {} -b {}'.format(
+                    "certs/wildcard.crt",
+                    "certs/wildcard.key",
+                    "certs/ca.crt",
+                ),
+                output_format='plain',
+            )
+        self.validate_output(result, cert_data)
+
+    @pytest.mark.parametrize("error, cert_file, key_file, ca_file", invalid_inputs)
```

Comment:
This doesn't really follow the established pattern of using `robottelo.datafactory` to define invalid inputs that can be parametrized for test functions, any reason for that?  Right now its only used for this one test function, but you defined the data structure at the top of the module, instead of parametrizing the four fields directly.  Are you planning on using invalid_inputs for future test coverage?

Reply:
Yes, I wanted to use for future test coverage this is raw structure, in case if it required to use multiple test functions across files I can update it to robottelo.datafactory